prime-router/src/main/kotlin/cli/helpers/HL7DiffHelper.kt

```diff
@@ -336,4 +346,27 @@ class HL7DiffHelper {
                 " Differences: $input$outputText"
         }
     }
+
+    /**
+     * helper
+     * Check that v1 and v2 are effectively both blank value
+     */
+    fun effectivelyBlank(v1: Array<ca.uhn.hl7v2.model.Type>, v2: Array<ca.uhn.hl7v2.model.Type>): Boolean {
```

Comment:
A few comments on this function:

1. Can we try a more descriptive variable name than `v1`/`v2`? Maybe something like `outputHL7v2Segment`/`inputHL7v2Segment`?
2. It might make sense to make this an `internal` function
3. Would it make sense to have the logic here be something more like: `( ( v1.size == 1 && v1[0].isEmpty ) || v1.isEmpty) && ( ( v2.size == 1 && v2[0].isEmpty ) || v2.isEmpty)`? Especially in isolation, it's a bit confusing that the logic for one Array being empty is different than the logic for the second array, so changing the logic up to be more commutative could be helpful.